### PR TITLE
Feature/setup invitations

### PR DESCRIPTION
**Pull request by mspector**
this is a script to setup UAI official reviews and comments.

It's very close to being generalizable, but in the short term, I need to run this before I can refactor it into openreview-py.

### CODE REVIEW

File: venues/auai.org/UAI/2018/process/officialReviewProcess.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      var author_mail = {
+        "groups" : authors,
+        "subject": "Review posted to your submitted paper: \"" + forum.content.title + "\"",
+        "message": "Your submissions to " + conference + " has received an official review. \n\nTitle: " + note.content.title + "\n\nComment: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum
+      };
+
+      var reviewer_mail = {
+        "groups" : reviewers,
+        "subject": "Review posted to your assigned paper: \"" + forum.content.title + "\"",
+        "message": "A submission to " + conference + ", for which you are an official reviewer, has received a review. \n\nTitle: " + note.content.title + "\n\nReview: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum
```
Comment (melisabok) [inline venues/auai.org/UAI/2018/process/officialReviewProcess.js]: I remembered to get some feedback asking to see the review content in the email.

Comment (mspector) [reply venues/auai.org/UAI/2018/process/officialReviewProcess.js]: oops, it's actually there already: note.content.review

Comment (mspector) [reply venues/auai.org/UAI/2018/process/officialReviewProcess.js]: I'm confused - the review content is already in the email. Am I misunderstanding what you mean? 

"message": "A submission to " + conference + ", for which you are an official reviewer, has received a review. \n\nTitle: " + note.content.title + "\n\nReview: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum

Comment (melisabok) [reply venues/auai.org/UAI/2018/process/officialReviewProcess.js]: Ohhh you are right! I'm sorry.

**Comment by mspector** [comment]
@melisabok @pmandler 

FYI: this branch is the most current one for controlling UAI comments and reviews

**Pull request merged**